numba/core/ir_utils.py

```diff
@@ -1857,11 +1864,14 @@ def gen_np_call(func_as_str, func, lhs, args, typingctx, typemap, calltypes):
     np_assign = ir.Assign(np_call, lhs, loc)
     return [g_np_assign, attr_assign, np_assign]
 
+def dump_block(label, block):
+    print(label, ":")
+    for stmt in block.body:
+        print("    ", stmt)
+
 def dump_blocks(blocks):
```

Comment:
At present this debug routine is used solely when `NUMBA_DEBUG_ARRAY_OPT` is set, it continues to work as expected after the split.